### ISSUE TITLE
UX - ctrl - K should remove connections of multiple highlighted objects

### ISSUE THREAD

**Issue by muziker** [pure-data/pure-data#865]
Currently ctrl-k removes all connections of one highlighted object. ctrl-k should remove connections of multiple highlighted objects.

**Comment by porres** [comment]
hey, shouldn't control+k disconnect just 2 objects? Like it connects them?

**Comment by muziker** [comment]
yes you're right it does do that. Maybe ctrl-shift-k should remove all connections of multiple objects

**Comment by umlaeute** [comment]
i don't know.

- how often are you (and others) perplexed because of the current behaviour?
- how often do you actually (and others) need this feature? 

(i try to not implement features just for the sake of them)

**Comment by porres** [comment]
> yes you're right it does do that.

it doesn't for me. If I have 2 objects connected and would like to disconnect them with control+k it doesn't work.

> how often are you (and others) perplexed because of the current behaviour?
> how often do you actually (and others) need this feature?

Anyway, sorry for hijacking this thread, but I thought I'd bring my issue up here as well. This is actually something that bothers me every now and then but not enough to stop patching and open an issue. More than that, I had the idea that this was working first time intelligent patching came around and then stopped working in the last version, or was it all a dream?

I have to confess I'm perplexed as I'd expect control+k to connect and disconnect in the same way, but now it connects 2 objects but doesn't disconnect them. I just tested this on mac

**Comment by muziker** [comment]
The behaviour of patching:

1.1 highlight one object
1.2 ctrl+k : disconnect all connections

2.1 highlight two objects:
2.2 ctrl+k: connect outlet to inlet
2.3 successive ctrl+k: connects successive outlets to inlets
